api/tests/opentrons/protocol_api/test_module_context.py

```diff
@@ -163,6 +164,13 @@ def test_load_labware_from_definition(
     decoy.verify(mock_core_map.add(mock_labware_core, result), times=1)
 
 
+def test_parent(decoy: Decoy, mock_core: ModuleCore, subject: ModuleContext) -> None:
+    """Should get the parent slot name."""
+    decoy.when(mock_core.get_deck_slot()).then_return(DeckSlotName.SLOT_1)
+
+    assert subject.parent == DeckSlotName.SLOT_1
```

Comment:
Make sure this method is returning a plain string

```suggestion
    assert subject.parent == "1"
    assert not isinstance(subject.parent, DeckSlotName)
```

Reply:
Because `DeckSlotName` inherits from `str`, an `== "1"` check isn't sufficient to make sure the subject is returning a plain string.

```python
>>> from enum import Enum
>>> class MyEnum(str, Enum):
...     MY_VALUE = "abc"
...
>>> MyEnum.MY_VALUE == "abc"
True
>>> isinstance(MyEnum.MY_VALUE, str)
True
```

If this is important, I would do something like:

```python
assert type(subject.parent) == str
assert subject.parent == "1"
```